e2e/page-objects/patient/patient-detail/patient-dialog.po.ts

```diff
@@ -0,0 +1,66 @@
+import { by } from 'protractor';
+import { BasePage } from '../../base-page';
+import { Button, Grid, InputField, Tabs } from '../../../widgets';
+
+export class PatientDetailPage extends BasePage {
```

Comment:
The name for this file must be "patient-detail.po.ts" instead of "patient-dialog.po.ts"